osquery/logger/logger.h

```diff
@@ -9,9 +9,11 @@
 #pragma once
 
 #ifdef WIN32
+#ifndef GLOG_NO_ABBREVIATED_SEVERITIES
```

Comment:
I'm not sure why this is necessary when there is already a `#pragma once`, but without it we get tons of warnings about the redefinition of this flag.

Reply:
That's because it's already defined here https://github.com/google/glog/blob/96a2f23dca4cc7180821ca5f32e526314395d26a/CMakeLists.txt#L504, so when everything is built from source, it will be defined though compiler invocation.

Reply:
Does that mean we are best off removing the #define here?

Reply:
Right, a possible alternative is to remove it from there and pass it through CMake also here https://github.com/osquery/osquery/blob/master/libraries/cmake/facebook/glog/CMakeLists.txt, since we still want to keep the ability to build with pre-built libraries.

Reply:
I'm happy to merge after we've compared the change in this PR against the suggestion in the discussion here.